Formula/marp-cli.rb

```diff
@@ -0,0 +1,50 @@
+class MarpCli < Formula
+  desc "CLI interface for Marp and Marpit based converters"
+  homepage "https://marp.app"
+  url "https://github.com/marp-team/marp-cli/releases/download/v1.1.1/marp-cli-v1.1.1-mac.tar.gz"
```

Comment:
We need to build from source, not use the precompiled binary.

Reply:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae